server/utils/decorator.py

```diff
@@ -61,28 +63,67 @@ def wrapper(*args, **kwargs):
 
             kwargs[arg] = value
             return f(*args, **kwargs)
+
         return wrapper
+
     return decorator
 
 
 def admin_only(func: Callable):
```

Comment:
This function now seems like a repetition of `role_required`. IMO this should be removed.